File: C/pset6/mario.py
from cs50 import get_int

def main():
    print("Enter a number!!",end=" :")
    A = get_int()

    for i in range(A):
        for j in range(i + 1):
            print("#",end="")

        print()


if __name__ == "__main__":
    main()

File: C/pset1/Mario/mario.c
#include <stdio.h>

int main(){
    int A,i,j;
    printf("enter a number!!");
    scanf("%d",&A);
        for(i=0;i<A;i++){
            for(j=0;j<=i;j++){
                printf("#");
            }
            printf("\n");
        }
}